resources/metadata/cnbBuild.yaml

```diff
@@ -138,6 +138,21 @@ spec:
           - PARAMETERS
           - STAGES
           - STEPS
+      - name: expandBuildEnvVars
+        type: "bool"
+        default: false
+        description: |
+          Replace placeholder with environment variables set by `buildEnvVars`.
```

Comment:
It's rather that this expands placeholders in the values of buildEnvVars using the environment `piper cnbBuild` runs in, right? Maybe we should ask Jen for the right wording.